packages/react-i18n/src/i18n.ts

```diff
@@ -227,6 +262,19 @@ export default class I18n {
       });
     }
   }
+
+  private decimalSymbol(currencyCode: string) {
+    const {symbol} = this.getCurrencySymbolLocalized(this.locale, currencyCode);
+    const templatedInput = Number('1000.00');
+    const formattedInput = this.formatCurrency(templatedInput, {
+      currency: currencyCode,
+    }).replace(symbol, '');
+    const decimal = formattedInput.charAt(formattedInput.length - 3);
```

Comment:
As Nima mentioned about toFixed(2), this won't work either for some currencies for example:
`new Intl.NumberFormat('ar', { style: 'currency', currency: 'JOD'}).format('1000.00');` => `د.أ.‏ 1,000.000`

Reply:
This also won't work for JPY because there is no decimal delimiter for that currency.
What about something like this:
- Use `templatedInput = 1`, not 1000 to avoid edge cases and
```js
new Intl.NumberFormat('en', { style: 'currency', currency: currencyCode})
  .format('1')
  .replace('1', '')
  .replace(currencyCode, '')
  .replace(/\s/, '')
```

And then get the first character?

- For JPY it should be undefined
- For JOD, it should be `.` since you'll have `.000` left.

Does it make sense?

We might want to try with arabic as well because the currency code will be `right-to-left` which will place the `.` at the right of the decimals instead of at the left... 🤦‍♂️

Reply:
True, we went a bit far when we can just use the JSON file with the data since we have the currency code... Thanks for bringing us back to the light Clauderic 🙏

Reply:
😞 I had a feeling. I would really like to use `Intl.NumberFormat.formatToParts` instead but can't seem to find a polyfill for it.

Reply:
> So you may need to guess the decimal mark after all :/

"guessing" the decimal was really my least preferred choice.

Reply:
Oh true...
But looking at the code, it seems that you use that method only to see if the currency supports decimal, so why not have a function that just does that?
Be like: 

```js
function supportsDecimal(currencyCode) {
  returns !CURRENCIES_WITH_NO_DECIMALS.includes(currencyCode)
}
```